src/linelast_solver.cpp

```diff
@@ -417,6 +418,28 @@ void LinElastSolver::SetupDomainBCOperators()
    }
 }
 
+void LinElastSolver::SetParameterizedProblem(ParameterizedProblem *problem)
+{
+   // Set lambda and mu coeffs
+   Vector lambda(numBdr);
+   Vector mu(numBdr);
```

Comment:
As far as I understand, this lame constants depend on the material property, which will vary according to domain attribute, not boundary attribute. Shouldn't this be a size of `numSub`?

While the first parameterized problem only supports constant `lambda` and `mu` over all domain, we should be able to set various types of material property. Since we are already using domain attribute for domain decomposition, we'll have to discuss how we would implement spatially-varying lame constants.

Regardless, `lambda` and `mu` don't seem to be sized with `numBdr`.